pkg/kv/kvserver/scheduler.go

```diff
@@ -168,6 +168,7 @@ type raftScheduler struct {
 	processor      raftProcessor
 	latency        *metric.Histogram
 	numWorkers     int
+	cordoned       map[roachpb.RangeID]bool
```

Comment:
This might be better as a new state for `replica.mu.destroyStatus`:

https://github.com/cockroachdb/cockroach/blob/84e3fdf8e7f9a4e7ec942e28cc17d2d4802cf74a/pkg/kv/kvserver/replica_destroy.go#L31-L40

Reply:
I agree, this state should be on the replica. Don't know all the implications of `destroyStatus` off the bat.

Reply:
I'll check this out!